TestWebAppProject/TestWebAppProject.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp1.1.1</TargetFramework>
```

Comment:
1111111

Reply:
i think this version should be different